### DISCUSSION TITLE
How can I make my own translation ?

### DISCUSSION THREAD

**Discussion by Benji59173** [YimMenu/YimMenu#996] [Q&A] [answered]
I didn't saw something on the Github to create my own translation (French Translation) ?
Is there any local folder where i can edit or make a new Translation ?
Thanks for answer !

**Comment by GeopJr** [comment, marked as answer]
Before proceeding, please read: https://github.com/YimMenu/Translations/issues/3

With that said:

- fork https://github.com/YimMenu/Translations/
- duplicate `en_US.json` and name it `<language_code>.json` (see: https://www.ibm.com/docs/en/radfws/9.6.1?topic=overview-locales-code-pages-supported for language codes) (`fr_FR.json` for french)
- translate all strings
- add your language in https://github.com/YimMenu/Translations/blob/master/index.json following the way the others are listed
```json
"fr_FR": {
  "name": "French",
  "file": "fr_FR.json"
}
```
- open a PR
